src/adapters/shell.coffee

```diff
@@ -12,11 +15,7 @@ historySize = if process.env.HUBOT_SHELL_HISTSIZE?
 
 class Shell extends Adapter
   send: (envelope, strings...) ->
-    unless process.platform is 'win32'
-      console.log "\x1b[01;32m#{str}\x1b[0m" for str in strings
-    else
-      console.log "#{str}" for str in strings
-    @repl.prompt()
+    console.log chalk.green.bold("#{str}") for str in strings
```

Comment:
It should, yeah. It uses the `supports-color` module for determining if the terminal supports color or not.